dnase/callsnpsMerge.sh

```diff
@@ -77,12 +78,17 @@ rm -f ${covfiles}
 
 unstarch ${sampleOutdir}/${name}.${mappedgenome}.coverage.starch | cut -f1-3,5 > $TMPDIR/${name}.${mappedgenome}.coverage.bedGraph
 
-#Fix problems with reads running off end of chromosome
-bedClip $TMPDIR/${name}.${mappedgenome}.coverage.bedGraph ${chromsizes} $TMPDIR/${name}.${mappedgenome}.coverage.clipped.bedGraph
+if [[ "${analyzedReads}" == 0 ]]; then
+    bedtools genomecov -bg -i <(echo -e "fakeChrom\t0\t1\tfakeReadID\t0\t+") -g <(echo -e "fakeChrom\t10") > $TMPDIR/${name}.${mappedgenome}.coverage.clipped.bedGraph
```

Comment:
how about just creating the dummy bedGraph file in the if statement, so the bedGraphToBigWig line is common to both?